ranker_trainer/LightGBMRankerTrainer/__init__.py

```diff
@@ -38,7 +52,13 @@ def train(self, *args, **kwargs):
         :param args: Additional arguments.
         :param kwargs: Additional key value arguments.
         """
-        # self.model = lgb.train(self.param, train_data, 2, valid_sets=[validation_data])
+        self.model = lgb.train(
```

Comment:
we need warm start, incremental training. This is training a new model each time right.